src/main/java/io/gatling/demostore/api/SwaggerConfig.java

```diff
@@ -1,67 +1,35 @@
+import io.swagger.v3.oas.models.OpenAPI;
+import io.swagger.v3.oas.models.info.*;
+import io.swagger.v3.oas.models.security.SecurityScheme;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
-import org.springframework.http.HttpMethod;
-import springfox.documentation.builders.ApiInfoBuilder;
-import springfox.documentation.service.ApiInfo;
-import springfox.documentation.service.AuthorizationScope;
-import springfox.documentation.service.Contact;
-import springfox.documentation.service.HttpAuthenticationScheme;
-import springfox.documentation.service.SecurityReference;
-import springfox.documentation.spi.DocumentationType;
-import springfox.documentation.spi.service.contexts.OperationContext;
-import springfox.documentation.spi.service.contexts.SecurityContext;
-import springfox.documentation.spring.web.plugins.Docket;
-
-import java.util.Collections;
-import java.util.List;
-
-import static springfox.documentation.builders.PathSelectors.regex;
 
 @Configuration
 public class SwaggerConfig {
 
-    @Bean
-    public Docket openApiDocket() {
-        return new Docket(DocumentationType.OAS_30)
-                .select()
-                .paths(regex("/api.*"))
-                .build()
-                .apiInfo(apiInfo())
-                .useDefaultResponseMessages(false)
-                .securitySchemes(Collections.singletonList(authenticationScheme()))
-                .securityContexts(Collections.singletonList(securityContext()));
-    }
-
-    private ApiInfo apiInfo() {
-        return new ApiInfoBuilder()
-                .title("Gatling DemoStore API")
-                .description("This is an API for a fictional / dummy eCommerce store that sells eyeglass cases. The create/update API endpoints only simulate writes, their results do not actually get persisted.")
-                .contact(new Contact("Gatling", "https://gatling.io/", null))
-                .version("1.0.0")
-                .build();
-    }
+    public static final String SECURITY_SCHEME = "bearerAuth";
 
-    private HttpAuthenticationScheme authenticationScheme() {
-        return HttpAuthenticationScheme
-                .JWT_BEARER_BUILDER
-                .name("JWT")
-                .build();
-    }
-
-    private SecurityContext securityContext() {
-        AuthorizationScope[] authorizationScopes = {new AuthorizationScope("global", "accessEverything")};
-        List<SecurityReference> securityReferences =
-                Collections.singletonList(new SecurityReference("JWT", authorizationScopes));
-        return SecurityContext.builder()
-                .securityReferences(securityReferences)
-                .operationSelector(this::isSecuredOperation)
-                .build();
-    }
-
-    private boolean isSecuredOperation(OperationContext context) {
-        return context.httpMethod() != HttpMethod.GET
-                && !context.getName().equals("createAuthenticationToken");
```

Comment:
No idea if it's still possible to define a global filter like this, but probably makes more sense anyway to use `@Operation(security = ...)` on the relevant routes.

Reply:
No, it's no longer possible. Local `@Operations` are the way to go. Then I didn't check how override was working. Does an annotation completely overrides? Or just the defined attributes?